### PR TITLE
Add a complete button & a job table to record the job status

### PR DESCRIPTION
**Pull request by Ji4chenLi**
This PR add a compete button on the fronted, and create a `Job` table to record the status of jobs.

### CODE REVIEW

File: simple-backend/nlpviewer_backend/lib/utils.py [hunk 1]
```diff
@@ -55,3 +55,23 @@ def fetch_project_check_perm(id, user, perm):
...
+        perm:
+            Permission to check. Example: "nlpviewer_backend.read_project"
+            
+    Returns:
+        A json response of the or forbidden or not found.
+        
+    """
+    job = get_list_or_404(Job, assignee=user)
+    
+    check_perm_project(job, user, perm)
```
Comment (hunterhector) [inline simple-backend/nlpviewer_backend/lib/utils.py:75]: I think the permission is not so correct. The `check_perm_project` checks if the user have the permission to the project, but here you provide a job?

**Review by hunterhector — COMMENTED** [review]
I think there is something not so right with the permission

File: simple-backend/nlpviewer_backend/lib/utils.py (hunk 1 above)
Comment (Ji4chenLi) [reply simple-backend/nlpviewer_backend/lib/utils.py:75]: Fixed

Comment (hunterhector) [reply simple-backend/nlpviewer_backend/lib/utils.py:75]: I am not sure if you can use `check_perm_project` function, see here:

https://github.com/asyml/stave/blob/master/simple-backend/nlpviewer_backend/lib/utils.py#L6

The first parameter should be a project not a job

**Review by hunterhector — APPROVED** [review]

**Pull request merged**